cost-analyzer/charts/grafana/templates/role.yaml

```diff
@@ -1,5 +1,4 @@
-{{ if .Values.global.grafana.enabled }}
-{{- if .Values.rbac.create }}
+{{ if and .Values.global.grafana.enabled .Values.rbac.create .Values.rbac.pspEnabled }}
```

Comment:
Is there any situation where we would want to create this role even when `.Values.rbac.pspEnabled=false`?

Reply:
I can't see how. Given that the Role appears to exist to service a single permission, and that permission seems to be PSP, why create an empty role if that isn't enabled?

Reply:
Yeah, I'm thinking the same. Let's move forward with this change!